manuscript/manuscript.md

```diff
@@ -528,7 +536,7 @@ of interest as outlined in Rule 9.
 National Center for Supercomputing Applications.
 Software Carpentry Foundation.
 iDigBio/NSF.
```

Comment:
Please add my funding info in: "NBZ was funded by the Gordon and Betty Moore Foundation through Grant GBMF 2550.03 to the Life Sciences Research Foundation."